packages/hooks/src/utils.ts

```diff
@@ -12,9 +12,14 @@ export function copyToSelf (target: any) {
       const getter = hasProtoDefinitions ? target.constructor.prototype.__lookupGetter__(key)
         : Object.getOwnPropertyDescriptor(target, key);
 
-      if (getter && hasProtoDefinitions) {
+      if (hasProtoDefinitions && getter) {
         target.__defineGetter__(key, getter);
-        target.__defineSetter__(key, target.constructor.prototype.__lookupSetter__(key));
+
+        const setter = target.constructor.prototype.__lookupSetter__(key);
+
+        if (setter) {
+          target.__defineSetter__(key, setter);
+        }
```

Comment:
This is because the `ctx.path` setter does not exist, feathers only offers a getter and it throws an error.